pkg/k8s/deployments/translate.go

```diff
@@ -246,3 +248,25 @@ func TranslateOktetoVolumes(spec *apiv1.PodSpec, rule *model.TranslationRule) {
 		spec.Volumes = append(spec.Volumes, v)
 	}
 }
+
+func translateSecurityContext(c *apiv1.Container, s *model.SecurityContext) {
```

Comment:
We should do something similar here:
https://github.com/okteto/okteto/blob/master/pkg/k8s/deployments/translate.go#L96
Check if it is defined in the pod or not, and only update the values that are not nil in dev.SecurityContext